chef/cookbooks/crowbar/recipes/default.rb

```diff
@@ -390,7 +390,7 @@
     realm: realm
   )
 
-  notifies :reload, resources(service: "apache2")
+  notifies :restart, resources(service: "apache2"), :immediately
```

Comment:
> What about just including the mod_* recipes before the apache2 one?

Tried that but doesn't work. The mod recipe includes the apache recipe which starts apache.